--- a/libero/libero/envs/env_wrapper.py
+++ b/libero/libero/envs/env_wrapper.py
@@ -12,11 +12,11 @@
 from io import BytesIO
 import cv2
 import ctypes
+from skimage.filters import gaussian
+from scipy.ndimage import zoom as scizoom
 try:
     from wand.api import library as wandlibrary
     from wand.image import Image as WandImage
-    from skimage.filters import gaussian
-    from scipy.ndimage import zoom as scizoom
 
     # Tell Python about the C method
     wandlibrary.MagickMotionBlurImage.argtypes = (ctypes.c_void_p,  # wand
@@ -109,7 +109,7 @@
     'mapsize' must be a power of two.
     """
     assert (mapsize & (mapsize - 1) == 0)
-    maparray = np.empty((mapsize, mapsize), dtype=np.float_)
+    maparray = np.empty((mapsize, mapsize), dtype=np.float64)
     maparray[0, 0] = 0
     stepsize = mapsize
     wibble = 100
